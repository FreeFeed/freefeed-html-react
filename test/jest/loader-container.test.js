/* global describe, it, expect */
import { render } from '@testing-library/react';

import LoaderContainer from '../../src/components/loader-container';

describe('AppUpdated', () => {
  it('Renders a full page loader', () => {
    const { asFragment } = render(
      <LoaderContainer loading fullPage>
        children
      </LoaderContainer>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
