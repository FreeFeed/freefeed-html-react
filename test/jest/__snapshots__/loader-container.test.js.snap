// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppUpdated Renders a full page loader 1`] = `
<DocumentFragment>
  <div
    class="loader-container -full"
  >
    <div
      class="loader-overlay"
    />
    children
  </div>
</DocumentFragment>
`;
